pod/annotations.go

```diff
@@ -626,3 +626,54 @@ func SidecarAnnotations(sidecarName, suffix string) []string {
 		LegacySidecarAnnotation(sidecarName, suffix),
 	}
 }
+
+type PlatformSidecar struct {
```

Comment:
you could start a `platform_sidecars.go` if you wanted.

Reply:
Yeah, the thought did cross my mind. Since right now this stuff is specific to annotations, and the annotation constants live in this file, it seemed reasonable to keep it in `annotations.go`, but I could go either way. Let's consider splitting it out to a separate file if the surface area grows beyond the few functions we have now.